### PR TITLE
Add ios_logging implementation module

### PR DESCRIPTION
**Pull request by trishnaguha**
Signed-off-by: Trishna Guha <trishnaguha17@gmail.com>

##### SUMMARY

ios_logging
net_logging test

##### ISSUE TYPE

 - Feature Pull Request
 
##### COMPONENT NAME

modules/network/ios/ios_logging
##### ANSIBLE VERSION

```
devel 2.4
```

### CODE REVIEW

File: lib/ansible/modules/network/ios/ios_logging.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+                    'facility': parse_facility(line),
+                    'level': parse_level(line, dest)})
+
+    return obj
+
+
+def map_params_to_obj(module):
+    obj = []
+
+    if 'collection' in module.params and module.params['collection']:
```
Comment (rcarrillocruz) [inline lib/ansible/modules/network/ios/ios_logging.py]: this should be aggregate now, change it here , argspec and docstrings

**Review by rcarrillocruz — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/ios/ios_logging.py [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+"""
+
+import re
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ios import get_config, load_config
+from ansible.module_utils.ios import ios_argument_spec, check_args
+
+
+def validate_size(value, module):
```
Comment (ganeshrn) [inline lib/ansible/modules/network/ios/ios_logging.py:111]: Seems like this function is not called from within the module.
You are missing the validate function that calls validation on each param.
Reference: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/junos/junos_vlan.py#L131-L137
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/junos/junos_vlan.py#L175

**Review by ganeshrn — CHANGES_REQUESTED** [review]
Comment added at L111 in ios_logging.py

**Comment by trishnaguha** [comment]
@rcarrillocruz @ganeshrn updated.

**Review by ganeshrn — APPROVED** [review]

**Comment by os4mike** [comment]
Many of the prior IOS versions only support  " | include " rather than  " | section " . In order to make my recent change work I've locally amended the map_config_to_obj function to replace the 'section' command with 'include' as when it comes to logging, their outputs are identical. Any thoughts on the matter for future releases?

**Pull request merged**